manifests/client.pp

```diff
@@ -54,9 +56,7 @@
   $conf_dir = $bacula::conf_dir
   $config_file = "${conf_dir}/bacula-fd.conf"
 
-  package { $packages:
-    ensure => present,
-  }
+  ensure_packages($packages,{ensure => $ensure})
```

Comment:
Minor style issue:
```suggestion
  ensure_packages($packages, {ensure => $ensure})
```

(Same for director and storage)

Reply:
Fixed.

Reply:
The Puppet Linter wanted more space, I've added them in the latest commit.